pkg/gen/node-templates/provider.ts.mustache

```diff
@@ -124,14 +124,14 @@ export namespace yaml {
      *        a. `{files: "foo.yaml", yaml: "(LITERAL YAML HERE)"}`
      */
     export class ConfigGroup extends CollectionComponentResource {
-        constructor(name: string, config: ConfigGroupOpts, opts?: pulumi.CustomResourceOptions) {
+        constructor(name: string, config: ConfigGroupOpts, opts?: pulumi.ComponentResourceOptions) {
             super("kubernetes:yaml:ConfigGroup", name, config, opts);
             this.resources = parse(config, {parent: this});
         }
     }
 
     export class ConfigFile extends CollectionComponentResource {
-        constructor(name: string, config: ConfigOpts, opts?: pulumi.CustomResourceOptions) {
+        constructor(name: string, config: ConfigOpts, opts?: pulumi.ComponentResourceOptions) {
```

Comment:
Hmm, why does this show up as changed in the .mustache but not in the .ts file? Am I missing something?

Reply:
Fixed.